ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/text/v1/text/_cq_editConfig.xml

```diff
@@ -0,0 +1,178 @@
+<!--                                    jcr:primaryType="cq:Widget"-->
+<!--                                    collapsed="{Boolean}true"-->
+<!--                                    collapsible="{Boolean}true"-->
+<!--                                    inputValue="advanced"-->
+<!--                                    name="./linkdialog/cq:adhocLinkTrackingTab"-->
+<!--                                    title="Link tracking"-->
+<!--                                    xtype="dialogfieldset">-->
+<!--                                <items jcr:primaryType="cq:WidgetCollection">-->
+<!--                                    <enable-->
+<!--                                            jcr:primaryType="nt:unstructured"-->
+<!--                                            attribute="enabletracking"-->
+<!--                                            fieldDescription="override analytics framework settings."-->
+<!--                                            fieldLabel="Custom link tracking"-->
+<!--                                            name="./linkdialog/cq:adhocLinkTrackingEnableTracking"-->
+<!--                                            xtype="checkbox">-->
+<!--                                        <listeners-->
+<!--                                                jcr:primaryType="nt:unstructured"-->
+<!--                                                check="function(component){var dlg=component.findParentByType('rtelinkdialog');dlg.enableSCFields(component.checked);}"/>-->
+<!--                                    </enable>-->
+<!--                                    <events-->
+<!--                                            jcr:primaryType="nt:unstructured"-->
+<!--                                            attribute="adhocevents"-->
+<!--                                            fieldDescription="e.g.: event2, event7."-->
+<!--                                            fieldLabel="Include SiteCatalyst events"-->
+<!--                                            name="./linkdialog/cq:adhocLinkTrackingEvents"-->
+<!--                                            xtype="textfield"/>-->
+<!--                                    <evars-->
+<!--                                            jcr:primaryType="nt:unstructured"-->
+<!--                                            attribute="adhocevars"-->
+<!--                                            fieldDescription="e.g.: eVar1: pagedata.url, prop4: 'const'."-->
+<!--                                            fieldLabel="Include SiteCatalyst variables"-->
+<!--                                            name="./linkdialog/cq:adhocLinkTrackingEvars"-->
+<!--                                            xtype="textfield"/>-->
+<!--                                </items>-->
+<!--                            </linkAdvanced>-->
+<!--                        </linkAttributes>-->
+<!--                    </linkDialogConfig>-->
+<!--                </links>-->
+                <misctools
```

Comment:
ideally, they should be controlled by design dialog in alignment with sites. by default we can enable the options that were there in AF1.

Reply:
analyzing that in v2 as cq-desing-dialog is not working for us right now.